src/bia_bob/_machinery.py

```diff
@@ -45,13 +45,19 @@ def bob(line: str = None, cell: str = None):
     if Context.model is None:
         init_assistant()
 
-    user_input = combine_user_input(line, cell)
+    if line in Context.variables:
+        image = Context.variables[line]
```

Comment:
I think I get it now:
```
%%bob image1
what is in this image?
```

`line = image1` and then `Context.variables["image1"]` returns the actual image object, probably a numpy array?

I guess then the logic is not entirely fool-proof yet?

Let's say

```
a = 1  

%%bob a
how are you?
```

This would also try to sent `a` as an image, would it?

Reply:
ok, so we need to check if it's an image

Reply:
> if line in Context.variables means/does.

it checks if a variable with the given name exists